tests-with-coverage/Readme.md

```diff
@@ -0,0 +1,611 @@
+# Measuring the coverage of your tests
+
+Learn how to measure the test coverage of your application. This guide covers:
+
+* Measuring the coverage of your Unit Tests
+* Measuring the coverage of your Integration Tests
+* Separating the execution of your Unit Tests and Integration Tests
+* Consolidating the coverage for all your tests
```

Comment:
I would add that this does not support code coverage in native mode.

Reply:
Added